static/js/components/PodcastEpisodeCard.js

```diff
@@ -31,34 +31,35 @@ export default function PodcastEpisodeCard(props: Props) {
   return (
     <Card
       className="podcast-episode-card low-padding"
-      onClick={openEpisodeDrawer}
       persistentShadow={persistentShadow}
     >
-      <div className="left-col">
-        <div className="episode-title">
-          <Dotdotdot clamp={2}>{episode.title}</Dotdotdot>
-        </div>
-        <div className="podcast-name">{podcast.title}</div>
-        <div className="play-date-row">
-          <PodcastPlayButton episode={episode} />
-          <div className="date">
-            {moment(episode.last_modified).format(EPISODE_DATE_FORMAT)}
+      <button onClick={openEpisodeDrawer}>
```

Comment:
`<button />` elements can't contain other interactive elements like the `<a />` element within the `PodcastPlayButton` below: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button